cpg-neo4j/src/main/java/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -226,12 +250,13 @@ class Application : Callable<Int> {
      */
     @OptIn(ExperimentalPython::class, ExperimentalGolang::class, ExperimentalTypeScript::class)
     private fun setupTranslationConfiguration(): TranslationConfiguration {
-        assert(files.isNotEmpty())
-        val filePaths = arrayOfNulls<File>(files.size)
+        assert(exclusive.files.isNotEmpty())
```

Comment:
The variable name `exclusive` sounds misleading.

Reply:
What do you think about `mutuallyExclusiveParameters` instead of `exclusive`?